lib/Doctrine/Migrations/Tracking/TableUpdater.php

```diff
@@ -88,7 +89,7 @@ private function getFromTable() : Table
         // remove columns from the table definition that we don't care about
         // so we don't try to drop those columns
         foreach ($table->getColumns() as $column) {
-            if (in_array($column->getName(), $columnNames, true)) {
+            if (in_array(strtolower($column->getName()), $columnNames, true)) {
```

Comment:
How does this behave on a different naming strategy?

Reply:
@SenseException Maybe strtolower both side ? I know it's not the best choice. What you will recommend for that ?

Reply:
Maybe you should describe the problem first, that you want to fix, including an example. You can edit your summary above for that example and describe why names aren't in the correct case. This example may also work as a test, so it is also okay to put the example into a test, that will fail first, before it can be fixed in your PR.